lib/pf/Authentication/Source/LDAPSource.pm

```diff
@@ -346,6 +346,7 @@ sub match_in_subclass {
         foreach $condition (grep { $_->{'operator'} eq $Conditions::IS_MEMBER } @{$own_conditions}) {
             $value = escape_filter_value($condition->{'value'});
             $attribute = $entry->get_value($condition->{'attribute'});
+            $attribute = escape_filter_value($attribute);
```

Comment:
Since you are already in the code can you add a check if the attribute is undef.
``` $attribute = $entry->get_value($condition->{'attribute'}) // '' ```;
Then I will merge it.